templates/CRM/Case/Form/Selector.tpl

```diff
@@ -55,7 +55,8 @@
   {$row.case_recent_activity_type}<br />{$row.case_recent_activity_date|crmDate}{else}---{/if}</td>
     <td class="crm-case-case_scheduled_activity_type">{if $row.case_scheduled_activity_type}
   {$row.case_scheduled_activity_type}<br />{$row.case_scheduled_activity_date|crmDate}{else}---{/if}</td>
-    <td>{$row.action|replace:'xx':$row.case_id}{$row.moreActions|replace:'xx':$row.case_id}</td>
+    {* todo: Is moreActions ever populated in this context? *}
```

Comment:
I bet it is copy-pasta from Contact, searching only gives results for Contact, nothing else. I think can be cut as there is no code that assigns it.

Reply:
I can't see how it could possibly get from Contact $ContextMenu to Case $row - but trust your judgement, just my opinion.